ADR/0049-vsa-support.md

```diff
@@ -0,0 +1,69 @@
+# ADR-0049: Verification Summary Attestations for Release Policies
+
+Date: 2025-07-02
+
+## Status
+
+Proposed
+
+## Context
+
+We are facing several challenges with our current approach to Enterprise Contract (EC) verification
+and SDL checks:
+
+1. **Release-time EC verification performance**: Running EC at release time requires checking a
+   large number of attestations, which takes too long. Scaling to handle large applications like
+   OpenShift is proving challenging.
+2. **Pipeline arrangement limitations**: Currently we have to run all SDL checks in the build
+   pipeline, which limits our options on how to arrange pipelines and prevents us from potentially
+   moving some checks into integration tests.
+3. **Redundant verification of unchanged artifacts**: Pre-merge EC checks for large snapshots often
+   include one freshly built image plus a number of images that haven't changed, yet we redo the EC
+   check on all images.
+4. **Scalability concerns**: As our applications grow in complexity (like OpenShift snapshots), the
+   current approach becomes increasingly inefficient.
+5. **Release externally-built artifacts**: Our current policies for release pipelines assume
+   artifacts were built on Konflux. This inhibits ingestion and release of artifacts from other
```

Comment:
1. What is to use case for releasing artifacts built by other systems? I believe additional work would be needed to support this.

2. What is the meaning of "artifact built by konflux"? One can use the konflux tasks (without running a full konflux instance) and the policies should accept the build.